src/controllers/profile.js

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
+// import * as authenticate from '../middlewares/authenticate';
+
+let router = Router();
+
+// router.post('/', (request, response)=>{
+//   console.log('login');
+//   let data=request.body;
+//   console.log(data);
+//   response.json({data});
+// });
+
+router.get('/', (request, response)=>{
```

Comment:
* Proper spacing and indentation
* Use return in same line if there is only one statement
```js
router.get('/', (request, response) => response.json('User profile'));
```